native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/oracledb-native/serialization-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "types": [
+    {
+      "name": "oracle.sql.converter.CharacterConverter1Byte"
+    },
+    {
+      "name": "oracle.sql.converter.CharacterConverterJDBC"
+    }
+  ],
+  "lambdaCapturingTypes": [
```

Comment:
Lets remove `lambdaCapturingTypes` since this is empty

Reply:
Resolved in [11f67e6](https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/432/commits/11f67e69f81b345dd5cf117bfef50c9e05a8c53b)